tests/test_bindings.py

```diff
@@ -30,6 +30,9 @@
 from nacl.exceptions import BadSignatureError, CryptoError
 
 
+_BA = bytearray
```

Comment:
I'd prefer not to alias like this

Reply:
Removed the aliasing. This forced a reflow of the byte representation of the basepoint to please flake8.